### DISCUSSION TITLE
font renderer

### DISCUSSION THREAD

**Discussion by vtorri** [lexbor/lexbor#302] [Ideas]
https://typedesign.netlify.app/tdrenderer.html

**Comment by sp00ck** [comment]
Inconsolata or JetBrains Mono are better

**Comment by vtorri** [comment]
i'l not talking about font, but renderer of fonts

**Comment by sp00ck** [comment]
look at https://github.com/nothings/stb
and whole gui https://github.com/Immediate-Mode-UI/Nuklear
pity drawing directly are complicated in SDL2 , meybe use x11 or other library for main window. Meybe improve You a gui? 
for gui exist https://github.com/rxi/microui  but ugly licence

**Comment by vtorri** [comment]
stb and nuklear, i know. nuklear is slow when there are a lot of objects that are drawn
i didn't know microui

anyway, they are just examples, because @lexborisov does not want dependencies and wants to write everything from scratch, iirc

**Comment by sp00ck** [comment]
from scratch?
library graphics and system too?
sorry but public domain is a good licence

**Comment by vtorri** [comment]
@lexborisov has written a document parser for API documentation while doxygen exists, he wants to parse opentype fonts while freetype does it, same for unit tests (check liibrary exists)
